tests/test_admin.py

```diff
@@ -150,8 +154,12 @@ def test_duplicate_url_is_replaced(self):
         """
 
         urls = self.modeladmin.get_urls()
-        duplicate_url = [u for u in urls if '/duplicate/' in u.regex.pattern]
-        name_url = [u for u in urls if 'cms_pagecontent_duplicate' == u.name]
+        if GTE_DJ20:
+            duplicate_url = [u for u in urls if '/duplicate/' in u.pattern.regex.pattern]
+            name_url = [u for u in urls if 'cms_pagecontent_duplicate' == u.name]
+        else:
+            duplicate_url = [u for u in urls if '/duplicate/' in u.regex.pattern]
+            name_url = [u for u in urls if 'cms_pagecontent_duplicate' == u.name]
```

Comment:
A bit pedantic but valid for DRY, name_url could be outside of the condition. Whilst editing can you add a comment above the if django stating that this is for a fallback for < = django 1.11 which uses a different url structure.